etc/test-helper.js

```diff
@@ -1,6 +1,6 @@
-const expect = require('expect.js');
-const simple = require('simple-mock');
-const fs = require('fs');
+var expect = require('expect.js');
+var simple = require('simple-mock');
+var fs = require('fs');
```

Comment:
JSHint didn't like that this was using an ECMAScript 6 language feature without explicitly declaring support for ES6. Since they don't need to be `const`s, I reverted them to `var`s.